bcda/models/models.go

```diff
@@ -64,10 +64,22 @@ type ACO struct {
 	SystemID           string       `json:"system_id"`
 	AlphaSecret        string       `json:"alpha_secret"`
 	PublicKey          string       `json:"public_key"`
-	Blacklisted        bool         `json:"blacklisted"`
 	TerminationDetails *Termination `json:"termination"`
 }
 
+// Blacklisted returns bool based on TerminationDetails.
+func (aco *ACO) Blacklisted() bool {
+	if aco.TerminationDetails != nil {
+		if aco.TerminationDetails.BlacklistType == Involuntary || aco.TerminationDetails.BlacklistType == Voluntary {
+			return true
+		} else {
+			return false
```

Comment:
Clarifying question: How common is it to have a blacklist type of `limited`? And is that worth logging... and is it worth logging if blacklisted ACO's are attempting to make requests?

Reply:
Was this answered in the insights meeting?